packages/merkle-distributor/src/CloudflareKVHelper.ts

```diff
@@ -0,0 +1,92 @@
+// This file contains some helper functions to wrap cloudflare Key Value store.
+
+const nodeFetch = require("node-fetch");
+
+if (!process.env.CLOUDFLARE_ACCOUNT_ID || !process.env.CLOUDFLARE_NAMESPACE_ID || !process.env.CLOUDFLARE_TOKEN) {
```

Comment:
Interesting. So I originally actually did something akin to this. My original implementation actually just had them as param. It just seemed much cleaner like this as you don't need all this extra code and to pass all this stuff around. The way it is now worked out to be about a third as much code. I optimized for this over the other pattern just because it was cleaner.

I'm happy to change it though! can you explain what your main reasoning for the change is? do you just not like this pattern or feel that it is hard to test? For me, it just seemed a bit messier so I avoided it.

Reply:
i agree in general this is cleaner and less code than passing the vars into every function. 

i think my main problem with this pattern is that I look at these files as low level libraries, and this is not a standard way to configure a library.

its also not clear how you would ever instantiate multiple instances of a library configured this way. I dont think this is a problem for our use case, but its a weird side effect of configuring with envs this way. 

testability is another benefit, but these would be hard to test regardless and not my primary concern, though it would be nice if we had a standard pattern we could use to make it more testable. 

If someone was inspecting your higher level scripts to see how to use the libraries, it would not be obvious that there are mandatory env vars which need to be set in order to use it, unless going into the lib code or actually running it. Compare this to a configuration object that is passed in on instantiation, 1. its much more obvious and 2. this is pretty much how 99% of js libraries are done. 

that said, i dont know if it really matters that much, as all the issues ive outlined are mainly around making it more consumable for broader usage and dont affect functionality in your use case. this are things we can always kick the can on and address if it every becomes a problem.

Reply:
Thanks again for this feedback David. I've gone and followed the pattern you laid out here on both helpers in this PR. Please take a look and let me know what you think!